Mithril update 2023-09-27 (#255)
input-output-hk/cardano-updates: blog/2023-09-27-mithril.md

---
title: Mithril Team Update
slug: 2023-09-27-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> The Mithril team has released a new distribution [`2337.0`](https://github.com/input-output-hk/mithril/releases/tag/2337.0) which brings the following enhancements: support for the `zstandard` compression of snapshot archives, support for Cardano node version in the snapshot metadata, and support for recording snapshot download statistics in the aggregator.

Reviewer: ```suggestion
The Mithril team has released a new distribution [`2337.0`](https://github.com/input-output-hk/mithril/releases/tag/2337.0), which includes the following enhancements: support for `zstandard` compression of snapshot archives, support for the Cardano node version in snapshot metadata, and support for recording snapshot download statistics in the aggregator.
```

> They have completed the refactoring and standardizition of the errors in the Mithril nodes, and an [Architectural Decision Record](https://mithril.network/doc/adr/6) has been published on the documentation website. Additionally, they have kept working on adding Cloudflare protection to the infrastructure.

Reviewer: ```suggestion
They also completed the refactoring and standardization of the errors in the Mithril nodes and published an [Architectural decision record](https://mithril.network/doc/adr/6) on the documentation website. Additionally, they kept working on adding Cloudflare protection to the infrastructure.
```

> Finally, they have fixed a performance issue on the stress test tool for the aggregator, and they have made some improvements on the documentation for SPOs.

Reviewer: ```suggestion
Finally, the team fixed a performance issue on the stress test tool for the aggregator and made some improvements to the documentation for SPOs.
```
